x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_executed_details.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { Ecs } from '../../../../graphql/types';
+import { escapeQueryValue } from '../../../../lib/keury';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+const Details = styled.div`
+  margin-left: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+`;
+
+const MarginLeftFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+const MarginRightFlexItem = styled(EuiFlexItem)`
+  margin-right: 3px;
+`;
+
+export const DraggableAuditdExecutedElement = pure<{
+  id: string;
+  name?: string;
+  field: string;
+  value: string | null;
+  queryValue?: string | null;
+}>(({ id, name, field, value, queryValue }) =>
+  value != null ? (
+    <DraggableWrapper
+      dataProvider={{
+        and: [],
+        enabled: true,
+        id: escapeDataProviderId(`auditd-executed-element-${id}-${field}-${value}`),
+        name: name ? name : value,
+        excluded: false,
+        kqlQuery: '',
+        queryMatch: {
+          field,
+          value: escapeQueryValue(queryValue ? queryValue : value),
+        },
+      }}
+      render={(dataProvider, _, snapshot) =>
+        snapshot.isDragging ? (
+          <DragEffects>
+            <Provider dataProvider={dataProvider} />
+          </DragEffects>
+        ) : (
+          <>{value}</>
+        )
+      }
+    />
+  ) : null
+);
+
+export const AuditdExecutedDetails = pure<{ data: Ecs }>(({ data }) => {
+  const id = data._id;
```

Comment:
Optional - I was wondering if we can create a specific interface for `AuditdExecutedDetails` and have a function to format the data to this interface to look cleaner